2.1-setup/clustering.md

```diff
@@ -1,39 +1,59 @@
-# Clustering 
+# Clustering & Contexts
 
- > Note - This feature is only available on the Enterprise Edition of AxonServer
+ > Note - This feature is only available on the Enterprise Edition of Axon Server
 
-When running Axon Server in a licensed edition, you can set up a cluster of Axon servers. The servers run in active/active mode, so each node can receive and handle requests.
+Axon Server Enterprise Edition has the concept of contexts. In a single Axon Server cluster you can support multiple bounded contexts, where events are stored in its own set of files and 
+commands and queries are only routed within the bounded context. Each context is assigned to one or more nodes in the Axon Server cluster, a node can have more than one contexts.
+
+Within a context there is one member that is leader. When storing events, replication of the data is managed by the leader. For the client this is transparent as Axon Server will 
+automatically redirect the requests to the leader. Routing of commands and queries are done on all members of the context, no special role for the leader in this.  
+
+There is one special context in Axon Server, called __admin_, this is the context that handles all administration for the complete Axon Server cluster. All requests to change contexts, define application
+authorizations and users need to be sent to a node in the __admin_ context.        
```

Comment:
If for you is ok, I will add an issue to remember to manage Application and Roles in two different tables, for _admin_ and for specific contexts.